run/serializers.py

```diff
@@ -0,0 +1,69 @@
+from rest_framework import serializers
+from rest_framework.parsers import JSONParser
+from .models import Run
+from .models import FilePath
+from storage.models import File
+from storage.serializers import FileSerializer
+
+
+class FilePathSerializer(serializers.ModelSerializer):
+    file = FileSerializer(many=False)
+
+    # definition = FileDefinitionSerializer()
+
+    class Meta:
+        model = FilePath
+        fields = ('file',
+                  # 'definition',
+                  )
+
+
+class RunReportSerializer(serializers.ModelSerializer):
+    file_path_set = FilePathSerializer(many=True, read_only=True)
```

Comment:
Shouldn't this be changed similar to the way input is changed? We want it to return something like:
files: {sss: ssss}